lib/msal-browser/docs/v4-migration.md

```diff
@@ -161,6 +171,81 @@ See the [Configuration doc](./configuration.md#system-config-options) for more d
 
 The `onRedirectNavigate` parameter will *only be supported* from `Configuration` object going forward and is removed from `RedirectRequest` and `EndSessionRequest` objects. Please ensure to set it in msal config if you need to use it.
 
+### Consolidation of extra request parameters
+
+The following request parameters have been removed:
+
+-   `authorizePostBodyParams`
+-   `tokenBodyParameters`
+-   `tokenQueryParameters`
+
+In order to simplify extra request parameters, generic extra parameters should go in the new `extraParams` request option. When `extraParams` are set in a request, they will be sent on all token service calls in either the URL query string or the request body, depending on the `httpMethod` configured (default is `GET`) in the request. **To submit extra parameters that MUST go in the URL query string, `extraQueryParameters` is still available.**
```

Comment:
Suggestion: Add examples of parameters that **must** go in query string and note that if you're not sure it likely should go in `extraParameters`

Reply:
Done, thanks!